lars-reimann/api-editor
# feat(gui): usage based filters

Closes #496.

### Summary of Changes

* Add new filter `usages:>2` (different operators, arbitrary number)
* Add new filter `usefulness:>2` (different operators, arbitrary number)

### Testing Instructions

1. Run this:
```sh
cd api-editor
./gradlew run
```
2. Open `localhost:4280` in the browser
3. Load API data into the GUI
4. Load Usage data into the GUI (for that API)
5. Type different variations of the new filters into the input field. Use the help button to check the available options.
